### PR TITLE
[ci] update elpi to 1.13.1

### PR DESCRIPTION
**Pull request by gares**
https://github.com/math-comp/hierarchy-builder/pull/226
https://github.com/LPCIC/coq-elpi/pull/227

### CODE REVIEW

File: dev/ci/user-overlays/14111-gares-update-elpi.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+overlay elpi https://github.com/LPCIC/coq-elpi coq-master+1.9.5 14111
+overlay hierarchy-builder https://github.com/mathcomp/hierarchy-builder coq-master+1.1.0 14111
```
Comment (gares) [inline dev/ci/user-overlays/14111-gares-update-elpi.sh:2]:
```suggestion
overlay hierarchy_builder https://github.com/mathcomp/hierarchy-builder coq-master+1.1.0 14111
```

File: dev/ci/user-overlays/14111-gares-update-elpi.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+overlay elpi https://github.com/LPCIC/coq-elpi coq-master+1.9.5 14111
+overlay hierarchy-builder https://github.com/math-comp/hierarchy-builder coq-master+1.1.0 14111
```
Comment (SkySkimmer) [inline dev/ci/user-overlays/14111-gares-update-elpi.sh:2]:
```suggestion
overlay hierarchy_builder https://github.com/math-comp/hierarchy-builder coq-master+1.1.0 14111
```

**Comment by gares** [comment]
There we are, it looks ready

**Review by Zimmi48 — APPROVED** [review]
@coqbot merge now

@gares please take care of the overlays

**Comment by Zimmi48** [comment]
BTW, any idea what the macOS failure is about? It has been failing everywhere for the last 15 hours.

**Comment by gares** [comment]
no, but shouldn't we remove that job anyway?

**Comment by Zimmi48** [comment]
Oh, it doesn't use the platform yet?

**Pull request merged**